packages/paste-core/components/keyboard-key/src/KeyboardKeyContext.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+
+import { KeyboardKeyVariants } from "./KeyboardKeyGroup";
+
+export interface KeyboardCombinationState {
+  /**
+   * The aray of currently active keys
+   * @default []
+   * @type { string[] }
+   * @memberof KeyboardCombinationState
+   */
+  activeKeys?: string[];
+  /**
+   * Whether the keys should show the disbled state
+   * @default null
```

Comment:
question: should disabled default to false?

Reply:
I guess the default is undefined which I see is set to a null value in other areas. I will update to default to false

Reply:
makes sense, just nice to have a default for optional props